src/modules/px4iofirmware/mixer.cpp

```diff
@@ -287,19 +295,29 @@ mixer_tick(void)
 		/* mix */
 		mixed = mixer_mix_threadsafe(&outputs[0], &r_mixer_limits);
 
-		/* the pwm limit call takes care of out of band errors */
-		pwm_limit_calc(should_arm, should_arm_nothrottle, mixed, r_setup_pwm_reverse, r_page_servo_disarmed,
-			       r_page_servo_control_min, r_page_servo_control_max, outputs, r_page_servos, &pwm_limit);
+		/* if mixer_mix_threadsafe returns zero, it did nothing */
+		if (mixed != 0) {
```

Comment:
@kd0aij @LorenzMeier  I am not familiar with the mixer part of the system. I have some questions. 

What does it mean to not be mixed? 

The GH diff is hard to read - this is the change:
![image](https://cloud.githubusercontent.com/assets/1945821/23487465/67cf8a40-fe8a-11e6-9b94-0ce0c28ea74d.png)

It looks like prior to this change, a mixed = 0 clamped all the outputs. And then updated r_page_actuators[i]  to the clamped values.  With the change the the clamping and updating will not happen is that the desired effect?

Or would 

![image](https://cloud.githubusercontent.com/assets/1945821/23487714/0390526a-fe8c-11e6-8b94-864e4bf07673.png)

be a proper change?